config/locales/alchemy.it.yml

```diff
@@ -0,0 +1,924 @@
+    "Please Signup": "Prego registrati per modificare il tuo sito web."
+    "Please choose": "Prego scegli"
+    "Please enter a new password": "Prego inserisci una nuova password."
+    "Please enter your email address": "Prego inserisci il tuo indirizzo email."
+    "Please log in": "Prego autenticati."
+    "Please seperate the tags with commata": "* Prego separa i tag con delle virgole."
+    "Properties": "Proprietà"
+    "Publish page": "Pubblica pagina"
+    "Read the License": "Leggi la Licenza"
+    "Redirects to": "Effettua un redirect a"
+    "Reload Preview": "Ricarica anteprima"
+    "Remove item from clipboard": "Rimuovi elemento dagli appunti"
+    "Remove tag filter": "Rimuovi filtro tag"
+    "Remove this content": "Rimuovi questo contenuto"
+    "Reset Imagemask": "Ripristina la maschera delle immagini"
+    "Reset password instructions": "Ripristina le istruzioni per la password"
+    "Select all": "Seleziona tutto"
+    "Select an content": "Seleziona un contenuto"
+    "Select style": "Seleziona lo stile"
+    "Send reset instructions": "Invia istruzioni per il ripristino"
+    show_elements: "Mostra Elementi"
+    show_element: "Mostra elemento"
+    "Show childpages": "Mostra pagine figlie"
+    "Show clipboard": "Mostra appunti"
+    "Show picture infos": "Mostra informazioni immagine"
+    "Show trash": "Mostra cestino"
+    "Site successfully created": "Sito web creato con successo."
+    "Site successfully removed": "Sito web rimosso con successo."
+    "Site successfully updated": "Sito web aggiornato con sucesso."
+    "Size": "Grandezza"
+    "Sort pages": "Riordina pagine"
+    "Successfully added content": "%{content} aggiunto con successo"
+    "Successfully deleted content": "%{content} eliminato con successo"
+    "Successfully saved content position": "Posizione del contenuto salvata con sucesso"
+    "Tags": "Tag"
+    "These pictures could not be deleted, because they were in use": "Queste immagini non possono essere eliminate poiché erano in uso: %{names}"
+    "This page is locked": "Questa pagina è attualmente bloccata da %{name}"
+    "Title": "Titolo"
+    "Trash": "Cestino"
+    "User created": "Utente %{name} creato"
+    "User deleted": "Utente %{name} eliminato"
+    "User updated": "Utente %{name} aggiornato"
+    "Validation failed": "Validazione fallita"
+    "Version": "Versione"
+    "View File": "Mostra File"
+    "Visit page": "Visita pagina"
+    "Warning!": "Attenzione!"
+    content_definition_missing: "Attenzione: Manca la definizione del contenuto. Prego controlla il file elements.yml"
+    element_definition_missing: "ATTENZIONE! Manca la definizione dell'elemento. Prego controlla il file elements.yml."
+    page_definition_missing: "ATTENZIONE! Manca la definizione del layout della pagina. Prego controlla il file page_layouts.yml."
+    "Welcome to Alchemy": "Benvenuto su Alchemy"
+    "Who else is online": "Chi altro è online"
```

Comment:
since it's [followed by](https://github.com/AlchemyCMS/alchemy_cms/blob/10b0025a446fad5ab27ec63b5b4fd99262367207/app/views/alchemy/admin/dashboard/_users.html.erb#L2) `:` maybe we can use "Altre persone online", because "Chi altro è online" makes me feel to a question but without the question mark it sounds strange.